src/main/java/ecdar/presentations/SystemRootPresentation.java

```diff
@@ -40,12 +40,12 @@ private void initializeDimensions() {
         setLayoutX(root.getX());
         root.getXProperty().bind(layoutXProperty());
 
-        setLayoutY(Grid.TOOL_BAR_HEIGHT);
+        setLayoutY(40);

```

Comment:
Consider making the constants in this file a function of `Ecdar.CANVAS_PADDING`

Reply:
`TOOLBAR_HEIGHT` constant introduced